### PR TITLE
Refine AI review prompt

### PR DESCRIPTION
**Pull request by maggie-lou**

### CODE REVIEW

**Review by tyler-french — APPROVED** [review]

**Comment by tyler-french** [comment]
Should we maybe add a note to avoid comments that are overly opinionated or critical. For example, an obscure edge case is missing or requesting to add a test case for something trivial

File: enterprise/tools/agent_review/main.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ Rules:
 - Remove any footnote-style numeric references such as "(#1)" or "(#2)" from comment bodies and the summary — GitHub interprets these as issue/PR links.
 - Omit comments that only praise, affirm, or acknowledge a change without raising an actionable concern or suggesting an improvement (e.g. "good cleanup", "no callers found — safe to remove", "looks correct").
 - Avoid "introductory" or summary phrases with labels or categorizations like "Performance improvement: strings.Split allocates. Use strings.SplitSeq instead.". Just write the comment directly, like "strings.Split allocates. Use strings.SplitSeq instead."
+- The summary section should not summarize any of the individual comments. It should only explain the purpose of the diff and findings with no line reference.
```
[the remark this answers is not in the document]
Comment (maggie-lou) [reply enterprise/tools/agent_review/main.go]: Fixed line -> file

**Comment by maggie-lou** [comment]
> Should we maybe add a note to avoid comments that are overly opinionated or critical. For example, an obscure edge case is missing or requesting to add a test case for something trivial

Personally I'd rather see a comment about an edge case or a missing test and decide whether to pay attention to it, rather than have the bot not raise its concerns and we potentially miss a bug

Going to leave it as is for now, but if you have a specific pet peeve feel free to refine the prompt further
